src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -187,7 +187,18 @@
     <WriteLinesToFile
       File="$(IntermediateOutputPath)$(MSBuildProjectName).InputFiles.txt"
       Overwrite="true"
-      Lines="@(Content)" />
+      Lines="@(Content->Distinct())" />
```

Comment:
In `Sdk.props` at line `<Content Include="**/*.sql">` we Include **all** .sql files as Content to compile.
When we again specify 
```xml
<ItemGroup>
    <Content Include="Procedures\csp_Test.sql">
        <SuppressTSqlWarnings>71502</SuppressTSqlWarnings>
    </Content>
</ItemGroup>
```
We add this file to `Content` list again. So it contains two "links" for that file. To avoid this we must at first remove reference to file from `Content` list, and then include it. Like below.
```xml
<ItemGroup>
    <Content Remove="Procedures\csp_Test.sql">
    <Content Include="Procedures\csp_Test.sql">
        <SuppressTSqlWarnings>71502</SuppressTSqlWarnings>
    </Content>
</ItemGroup>
```

I think that this is too hard to remeber to specify "Remove" for every file. I prefer to simply copy section from `sqlproj` file
and remove duplicates from `Content` list

Reply:
That explanation makes sense to me! 👍

Reply:
I think you should be able to do:

```xml
<ItemGroup>
    <Content Update="Procedures\csp_Test.sql">
        <SuppressTSqlWarnings>71502</SuppressTSqlWarnings>
    </Content>
</ItemGroup>
```

which would avoid the issue. But I guess it can't hurt to do the Distinct as well :).

Reply:
Yea you can use `Update` clause, but because all editing make manually in large project with many files, folders etc. It became too difficult to monitor what file you must `Include`, and which you must `Update`